certified-connectors/QPP NG Integration Hub/apiDefinition.swagger.json

```diff
@@ -72,29 +72,30 @@
                 "newData": {
                   "type": "object",
                   "properties": {},
-				  "title": "New Data",
+                  "title": "New Data",
                   "description": "New data for a given event"
                 }
               },
-			  "title": "Additional Info",
+              "title": "Additional Info",
               "description": "additionalInfo"
             }
           }
         }
       },
       "post": {
-        "description": "Registers a webhook for a given event and Change type.",
+        "description": "Registers a webhook for a given event and Change type. For more information Check the github documentation. For more information , check the github documentation for QPP NG Integration Hub connector. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/QPP%20NG%20Integration%20Hub/ReadMe.md",
```

Comment:
Kindly keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

Reply:
![image](https://github.com/microsoft/PowerPlatformConnectors/assets/144774487/c4070950-ddb0-40ca-91c2-5e02148c2773)
Hello , I think the summaries were required to be short and under 80 characters. Not description as shared here in the certification process.

Reply:
hello @vmanoharas ,

As mentioned we have fixed issues with the description which were too long. 
But in case its still not as per requirement, please let us know what the exact character length for that is. As explicitly is also mentioned in the case of summaries.